playwright_test/tests/chatwidgetlaunch.spec.js

```diff
@@ -0,0 +1,9 @@
+import { test, expect } from "@playwright/test";
+import ChatWidgetPage from "../page/chatwidget.page";
+
+test("test", async ({ page }) => {
```

Comment:
I guess here, "test" will be the name of test

shouldn't we use something more suitable for identification?

Reply:
Have renamed the test.